packages/config/lib/index.js

```diff
@@ -101,4 +102,18 @@ module.exports = {
       }
     })
   },
+
+  validateNoReadOnlyConfig: (config, onErr) => {
+    let errProperty
+
+    Object.keys(config).forEach((element) => {
+      if (!writeableOptions.includes(element)) {
+        errProperty = element
+      }
+    })
```

Comment:
This will set errProperty to the last unwritable option; should we return a list of unwritable options if more than one are found?

If not, we can abort early with `.some`:
```js
Object.keys(config).some((option) => {  
  return errProperty = !writeableOptions[option] ? option : undefined
});
```

Reply:
Added `.some` for now, but can change this if we decide to return a list of options later on